lib/core/bloc/settings_cubit.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:bloc/bloc.dart';
+import 'package:kahootify/core/data/shared_preferences_repository.dart';
+import 'package:kahootify/core/models/settings.dart';
+
+class SettingsCubit extends Cubit<Settings> {
+  final SharedPreferencesRepository prefs;
+
+  SettingsCubit(this.prefs) : super(Settings.initial());
+
+  void initialize() {
+    final String? playerName = prefs.playerName;
+    final bool? enabledMusic = prefs.enabledMusic;
```

Comment:
it's way better to start the variable name from a noun. musicEnabled says everything, enabledMusic is like? hasUserEnabledMusic? Does android allow to play music?

Reply:
done